CONTRIBUTING.md

```diff
@@ -30,22 +30,23 @@ For more information see the [Code of Conduct FAQ](https://opensource.microsoft.
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
 ## Developing
-The simplest setup is:
+The simplest setup (assuming you have PyTorch already installed) is:
 ```
 mkdir hummingbird
 cd hummingbird
 git clone https://github.com/microsoft/hummingbird.git .
 pip install -e .[docs,tests,extra]
 ```
-On Windows, the last line above must also contain `-f https://download.pytorch.org/whl/torch_stable.html`. (This is required because PyTorch version for Windows is not up to date.)
+
+If you do not have pytorch installed, please [install it](https://pytorch.org/get-started/locally/) first.
```

Comment:
it seems `pip install torch==1.6.* -f https://download.pytorch.org/whl/torch_stable.html` should work in  MOST cases...do you want to suggest that here?  I'm fine  with also keeping this as a simple link :)

Reply:
It won't because you need to figure which cuda version to use if you want to go with gpu. Like +cu101. I think it is better to tell to just check the website.